<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>Chi tiết Chức Vụ</string>
   </property>
   <widget class="QPushButton" name="btnThemChucVu">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm Chức Vụ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSuaChucVu">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cập Nhật</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã Chức Vụ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tên Chức Vụ :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTenChucVu">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="txtMaChucVu">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MACV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hệ Số Lương :</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="txtHeSoLuong">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Phụ cấp :</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="txtPhuCap">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="TableChucVu">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>471</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
